using-valtimo/document/link-process-and-case.md

```diff
@@ -13,8 +13,8 @@ This page requires the following:
 
 To link a process to a case, the following steps are necessary:
 
-1. Go to the Admin menu.
-2. Go to the 'Cases' menu.
+1. Go to the menu 'Admin'.
```

Comment:
I'll keep pointing these out, and we can decide what to do with them later. To me, switching the order of admin and menu feels less natural. The intention is for it to be used as an adjective.